modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -220,3 +247,21 @@ function acquia_cms_site_studio_css_alter(array &$css) {
     unset($css[$path]);
   }
 }
+
+/**
+ * Function to add default settings of node revision delete.
+ */
+function _acquia_cms_site_studio_default_node_revision_delete_settings() {
+  // Install node revision delete module and set default configurations.
+  \Drupal::service('module_installer')->install(['node_revision_delete']);
+  $config = \Drupal::service('config.factory')->getEditable('node_revision_delete.settings');
+  if ($config) {
```

Comment:
@apathak18 Revisit this as `$config` will always returns TRUE, even in case of configuration not present